lib/pymedphys/tests/dicom/test_connect.py

```diff
@@ -234,10 +235,10 @@ def test_dicom_listener_cli(test_dataset):
         assert status.Status == 0
         assoc.release()
 
-    series_dir = HERE.joinpath(test_dataset.SeriesInstanceUID)
+    series_dir = test_directory.joinpath(test_dataset.SeriesInstanceUID)
     file_path = series_dir.joinpath(f"RP.{test_dataset.SOPInstanceUID}")
     read_dataset = pydicom.read_file(file_path)
     assert read_dataset.SeriesInstanceUID == test_dataset.SeriesInstanceUID
 
     # Clean up after ourselves
-    shutil.rmtree(series_dir)
+    shutil.rmtree(test_directory)
```

Comment:
I find using temp directory as a context manager makes for really nice clean code and this "rmtree" step is then handled appropriately by the standard library itself:

https://docs.python.org/3/library/tempfile.html#examples